pyproject.toml

```diff
@@ -0,0 +1,75 @@
+[build-system]
+requires = ["setuptools>=67", "setuptools_scm[toml]>=6.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "maraboupy"
+version = "1.0.0"
+authors = [{ name = "Wen Kokke", email = "wenkokke@users.noreply.github.com" }]
+description = "A package wrapping Marabou for Vehicle."
```

Comment:
Update the authors and the descriptions here?

Reply:
Where should I copy the description from?

Reply:
Assuming this is meant to be a short description, naybe just "A package wrapping the Marabou Neural Network Verification tool." ?  
Since I suppose this will be useful beyond Vehicle?

Reply:
Fixed.